guide/src/commands/relaying/clear.md

```diff
@@ -9,28 +9,26 @@ and [packet-acks](../raw/packet.md#relay-acknowledgment-packets).
 ### Usage
 
 ```
-Clear outstanding packets (i.e. packet-recv and packet-ack) on a given channel in both directions.
-
-The channel is identified by the chain, port, and channel IDs at one of its ends
-
 USAGE:
-    hermes clear packets <CHAIN_ID> <PORT_ID> <CHANNEL_ID>
+    hermes clear packets --chain <CHAIN_ID> --port <PORT_ID> --chan <CHANNEL_ID>
+
+DESCRIPTION:
+    Clear outstanding packets (i.e. packet-recv and packet-ack) on a given channel in both directions.
 
-ARGS:
-    <CHAIN_ID>      identifier of the chain
-    <PORT_ID>       identifier of the port
-    <CHANNEL_ID>    identifier of the channel
+    The channel is identified by the chain, port, and channel IDs at one of its ends
 
-OPTIONS:
-    -h, --help    Print help information
+FLAGS:
+        --chain <CHAIN_ID>     identifier of the chain
+        --chan <CHANNEL_ID>    identifier of the channel
+        --port <PORT_ID>       identifier of the port
 ```
 
 ### Example
 
 1. Without Hermes running, send 3 packets over a channel, here `channel-13`:
 
 ```
-❯ hermes tx raw ft-transfer ibc1 ibc0 transfer channel-13 9999 -o 1000 -n 3
+❯ hermes tx raw ft-transfer --dst-chain ibc1 --src-chain ibc0 --src-port transfer --src-chan channel-13 --amount 9999 --timeout-height-offset 1000 --number-msgs 3
```

Comment:
We still have `--src-chain` and `--dst-chain` here, not `--a-chain` and `--b-chain` as ADR 010 stipulates?

Reply:
Yes, these are the `raw` commands, which are not in the ADR 010 since they are used only by developers. So they have not been changed.